docs/visual-basic/programming-guide/language-features/procedures/snippets/how-to-call-an-event-handler/Class1.vb

```diff
@@ -0,0 +1,26 @@
+    '****************************************************************************
+    ' Special Form class to test Snippet5 -- changed Sub catchClose from Public to Protected 9/29/2005 for FxCop
+    ' Changed Protected catchClose to Private 4/6/2006 for FxCop -- jcooke
+    Public Class SpecialForm : Inherits System.Windows.Forms.Form
+        '<Snippet5>
+        ' Place these procedures inside a Form class definition.
+        Private Sub catchClose(sender As Object, e As System.ComponentModel.CancelEventArgs)
+            ' Insert code to deal with impending closure of this form.
+        End Sub
+        Public Sub formOpened()
+            AddHandler Me.FormClosing, AddressOf catchClose
```

Comment:
I changed Me.Closing, which is obsolete, to Me.FormClosing.

@adegeo @IEvangelist, Obviously, `catchClose` is a bad naming for the event. Have you settled on a convention?

Reply:
It is preferred to name it like this, `OnFormClosing`.